### PR TITLE
Fixed IE: Inputs with number type doesn't recognize

### LINKED ISSUE #3394: IE: Inputs with number type doesn't recognize
Fix other inputs that don't work with Numbers as expected

**Environment:** Development, Windows 10, IE 11.

"Add Proxy" form, EMQ item
imgpsh_fullsize 3

Branding page, colors block
imgpsh_fullsize 5

### PR DESCRIPTION
**Pull request by deligence-dharmendra**
Closes #3394

## Changes

- changes in EMQ and branding schema 
- add auto form type

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @marla-singer 

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/branding/collection/schema.js [hunk 1]
```diff
@@ -52,6 +52,9 @@ Branding.schema = new SimpleSchema({
     max: 100,
     optional: true,
     defaultValue: 0,
+    autoform: {
+      type: 'number',
```
Comment (marla-singer) [inline apinf_packages/branding/collection/schema.js]: Make the same changes for Settings -> email schema as well (That you changed on the previous PR)
https://github.com/apinf/platform/blob/develop/apinf_packages/settings/collection/schema.js#L84

Comment (deligence-dharmendra) [reply apinf_packages/branding/collection/schema.js]: Ok

**Comment by marla-singer** [comment]
@deligence-dharmendra Please test these inputs on nightly environment

**Comment by deligence-dharmendra** [comment]
@marla-singer I have tested the functionality in the nightly environment. it's working fine.

**Pull request merged**